src/content/api/auth.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Auth
+description: Introduction to Authentication
+draft: true
+nav:
+  path: API
+  order: 2
+---
+
+## Authenticating API Calls
+
+API calls can be authenticated by either providing an API key in the
+"X-Api-Key" header or by providing a user access token in the "Authorization"
+header.
+
+Org [Accounts](https://docs.centrapay.com/api/accounts) accessed with a user access token require the
```

Comment:
Are we doing absolute links while we do the migration?

Reply:
I think you'll have to use them until you get the file you are referencing into Astro. I suggest creating a ticket to change them to relative links when you are migrating.